src/microbe_stage/OrganismStatisticsPanel.cs

```diff
@@ -0,0 +1,629 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using Godot;
+
+/// <summary>
+///   Displays organism statistics calculated by an editor component
+/// </summary>
+public partial class OrganismStatisticsPanel : PanelContainer
+{
+    [Export]
+    public bool ShowHealthStat;
+
+    [Export]
+    public bool ShowSizeStat;
+
+    [Export]
+    public bool ShowStorageStat;
+
+    [Export]
+    public bool ShowSpeedStat;
+
+    [Export]
+    public bool ShowRotationSpeedStat;
+
+    [Export]
+    public bool ShowDigestionSpeedStat;
+
+    [Export]
+    public bool ShowDigestionEfficiencyStat;
+
+#pragma warning disable CA2213
+
+    [Export]
+    public LabelSettings ATPBalanceNormalText = null!;
+
+    [Export]
+    public LabelSettings ATPBalanceNotEnoughText = null!;
+
+#pragma warning restore CA2213
+
+    private readonly StringBuilder atpToolTipTextBuilder = new();
+
+    private LightLevelOption selectedLightLevelOption = LightLevelOption.Current;
```

Comment:
In my style I prefer to put normal variables after the godot ones, so I'd move this variable down a bunch.